meta-zephyr-sdk/recipes-devtools/qemu/zephyr-qemu_git.bb

```diff
@@ -5,7 +5,7 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 LIC_FILES_CHKSUM = "file://COPYING;md5=441c28d2cf86e15a37fa47e15a72fbac \
                     file://COPYING.LIB;endline=24;md5=8c5efda6cf1e1b03dcfd0e6c0d271c7f"
 
-SRCREV = "609d7596524ab204ccd71ef42c9eee4c7c338ea4"
+SRCREV = "44f28df24767cf9dca1ddc9b23157737c4cbb645"
 SRC_URI = "git://github.com/qemu/qemu.git;protocol=https \
 	   https://github.com/zephyrproject-rtos/seabios/releases/download/zephyr-v1.0.0/bios-128k.bin;name=bios-128k \
```

Comment:
Seems like some of these patches should drop as I think they got back ported from qemu-6.2.

Should we move go zephyr git repo for qemu?

Reply:
Updated to use the [`zephyr-qemu-v6.2.0` branch](https://github.com/zephyrproject-rtos/qemu/tree/zephyr-qemu-v6.2.0) in the Zephyr QEMU fork repository.